card/db/migrate_core_cards/data/more_bootstrap.json

```diff
@@ -0,0 +1,51 @@
+      },
+      {
+        "name": "simple skin",
+        "type": "Skin",
+        "content": "[[style: jquery-ui-smoothness]]\n[[style: functional]]\n[[style: standard]]\n[[style: right sidebar]]\n[[style: common]]\n[[style: traditional]]\n[[style: glyphicons]]\n[[style: bootstrap compatible]]"
+      },
+      {
+        "name": "simple bootstrap skin",
+        "type": "Skin",
+        "content": "[[style: jquery-ui-smoothness]]\n[[style: functional]]\n[[style: standard]]\n[[style: right sidebar]]\n[[style: common]]\n[[style: bootstrap]]\n[[style: bootstrap theme]]\n[[style: bootstrap cards]]"
+      },
+      {
+        "name": "Default Layout",
+        "type": "Layout",
+        "content": "<!DOCTYPE HTML>\r\n<html>\r\n  <head>\r\n    {{*head}}\r\n  </head>\r\n  \r\n  <body>\r\n    <div id=\"menu\">\r\n      {{*page header}}\r\n    </div>\r\n    \r\n    <div id=\"primary\">\r\n      {{_main}}\r\n    </div>\r\n    \r\n    <div id=\"secondary\">\r\n      {{*sidebar}}\r\n    </div>\r\n  </body>\r\n</html>"
+      },
```

Comment:
I love this.  worth discussing how much it would break to make the names and classes line up, but this is wonderfully simple!